### PR TITLE
SITL: add Simulink rover example

### PR DESCRIPTION
**Pull request by IamPete1**
This adds a Simulink rover example along with a library for the required ArduPilot connection blocks.

Annoyingly Simulink files are versioned, these will only work on 2020a (or newer). I do have the option to back date for older versions but they make no guarantee that everything will still work.

The rover block used in this example requires the Robotics System Toolbox.

https://youtu.be/hTFyMrjwQlI

### CODE REVIEW

File: libraries/SITL/SIM_JSON.cpp [hunk 1]
```diff
@@ -273,47 +276,42 @@ void JSON::recv_fdm(const struct sitl_input &input)
 // @Field: GY: Simulated gyroscope, Y-axis
 // @Field: GZ: Simulated gyroscope, Z-axis
     AP::logger().Write("JSN1", "TimeUS,TUS,R,P,Y,GX,GY,GZ",
-                       "QQffffff",
+                       "Qfffffff",
```
Comment (tridge) [inline libraries/SITL/SIM_JSON.cpp:279]: add units

File: libraries/SITL/SIM_JSON.cpp [hunk 2]
```diff
@@ -232,6 +232,9 @@ void JSON::recv_fdm(const struct sitl_input &input)
                            state.velocity[1],
                            state.velocity[2]);
 
+    airspeed = velocity_ef.length();
```
Comment (tridge) [inline libraries/SITL/SIM_JSON.cpp:235]: needs to be on x-axis

File: libraries/SITL/SIM_JSON.cpp [hunk 3]
```diff
@@ -232,6 +232,9 @@ void JSON::recv_fdm(const struct sitl_input &input)
...
+    airspeed_pitot = airspeed;
```
Comment (IamPete1) [inline libraries/SITL/SIM_JSON.cpp:236]: fix this to get along the x axis

**Comment by IamPete1** [comment]
Added units, and fixed airspeed calculation. Added Simulink to readme along with the demo vids.

**Pull request merged**